adapter/internal/oasparser/model/mgw_swagger.go

```diff
@@ -412,6 +414,37 @@ func (swagger *MgwSwagger) SetXWso2Extensions() error {
 	return nil
 }
 
+// SetRateLimitPoliciesForOperations assings rate limit policies to work with envoy rate limit service
+func (swagger *MgwSwagger) SetRateLimitPoliciesForOperations(apiYamlOperations []OperationYaml) {
+	if swagger.RateLimitLevel == "API" {
+		for _, resource := range swagger.resources {
+			for _, operation := range resource.methods {
+				operation.RateLimitPolicy = swagger.RateLimitPolicy
+			}
+		}
```

Comment:
I think we do not need this part. If the policy is API level set it in each operation.